scripts/win32.bat

```diff
@@ -107,8 +127,8 @@ For Each TopLevelDrive In GetTopLevelDrives()
 		Wscript.Echo "mountpoints: []"
 	Else
 		Wscript.Echo "mountpoints:"
-		For Each Mountpoint In TopLevelDrive.Item("Mountpoints")
-			Wscript.Echo "  - path: """ & Mountpoint & """"
+		For Each Mountpoint In TopLevelDrive.Item("Mountpoints").GetArray()
+			Wscript.Echo "	- path: """ & Mountpoint & """"
```

Comment:
Ugh, the space was replaced with a tab, which causes YAML to break :/ I'll fix in a second.